### PR TITLE
add Performance to System Tools

### PR DESCRIPTION
**Pull request by rembiesa**

### CODE REVIEW

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #16046 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/16046)

File: languages/en-US/_Base.json [hunk 1]
```diff
@@ -482,6 +482,7 @@
     "LBL_SUMMARY_DETAILS": "Summary details",
     "LBL_Sun": "Sun",
     "LBL_SUPPORT": "SUPPORT",
+    "LBL_PERFORMANCE": "Performance",
```
Comment (rskrzypczak) [inline languages/en-US/_Base.json]: Move all labels to correct file (\Settings\ConfigEditor.json)

File: modules/Settings/ConfigEditor/models/Module.php [hunk 2]
```diff
@@ -30,6 +30,10 @@ class Settings_ConfigEditor_Module_Model extends Settings_Vtiger_Module_Model
 		'langInLoginView' => 'LBL_SHOW_LANG_IN_LOGIN_PAGE',
 		'layoutInLoginView' => 'LBL_SHOW_LAYOUT_IN_LOGIN_PAGE',
 	];
+	/** {@inheritdoc} */
```
Comment (rskrzypczak) [inline modules/Settings/ConfigEditor/models/Module.php]: inherited after what?

**Review by rskrzypczak — CHANGES_REQUESTED** [review]

File: layouts/basic/modules/Settings/ConfigEditor/Detail.tpl [hunk 3]
```diff
@@ -28,6 +33,9 @@
 			<div class="js-tab tab-pane {if $ACTIVE_TAB eq 'Relation'}active{/if}" id="Relation" data-name="Relation" data-js="data">
 				{include file=\App\Layout::getTemplatePath('Relation.tpl', $QUALIFIED_MODULE)}
```
Comment (rskrzypczak) [inline layouts/basic/modules/Settings/ConfigEditor/Detail.tpl]:
```suggestion
				{include file=\App\Layout::getTemplatePath('ConfigTamplate.tpl', $QUALIFIED_MODULE) CONFIG_NAME='Relation'}
```

**Pull request merged**